students/kbindhu/session06/weapon.py

```diff
@@ -0,0 +1,37 @@
+def score(weapon_type,weaponsize):
+    score_val=weapon_type(weaponsize)
+    return score_val
+
+
+"""hand weapon"""
```

Comment:
This looks like a doc string. If you put it just inside the function definition you gain access to it at runtime via the __doc__ attribute.
https://www.python.org/dev/peps/pep-0257/
https://en.wikipedia.org/wiki/Docstring#Python